flytekit/annotated/workflow.py

```diff
@@ -128,12 +131,22 @@ def _local_execute(self, ctx: FlyteContext, **kwargs) -> Union[Tuple[Promise], P
         #   other things as well? What if someone just returns 5? Should we disallow this?
         function_outputs = self._workflow_function(**kwargs)
 
-        output_names = list(self.interface.outputs.keys())
-        if len(output_names) == 0:
-            if function_outputs is None:
-                return None
-            else:
-                raise Exception("something returned from wf but shouldn't have outputs")
+        output_names = list(self._interface.outputs.keys())
```

Comment:
Does this section work?  Where is output_literal_map used?

Reply:
huh, this is not changed by me, just i guess _interface changed to `interface`

Reply:
this is your code (old) I just added the instance check. That works :D But you are right output_literal_map does not seem to be used